dts/bindings/serial/nordic,nrf-uart-common.yaml

```diff
@@ -43,3 +43,27 @@ properties:
       description: |
         The CTS pin to use. The pin numbering scheme is the same as the
         tx-pin property's.
+
+    current-speed:
+      type: int
+      required: true
+      description: Initial baud rate setting for UART
+      enum: 
+        - 1200 # actual 1205
+        - 2400 # actual 2396
+        - 4800 # actual 4808
+        - 9600 # actual 9598
+        - 14400 # actual 14401
+        - 19200 # actual 19208
+        - 28800 # actual 28777
+        - 31250
+        - 38400 # actual 38369
+        - 56000 # actual 55944
+        - 57600 # actual 57554
+        - 76800 # actual 76923, 1.6% error, use with caution
+        - 115200 # actual 115108
+        - 230400 # actual 231884
+        - 250000 
+        - 460800 # actual 457143
+        - 921600 # actual 941176, > 2% error,use with caution
```

Comment:
Comments like this are more useful if you put them in a multi-line description, so they show up in the bindings index pages for these: https://docs.zephyrproject.org/latest/reference/devicetree/bindings/nordic%2Cnrf-uarte.html#nordic-nrf-uarte